ci-operator/step-registry/single-node/add-worker/2/single-node-add-worker-2-ref.yaml

```diff
@@ -0,0 +1,13 @@
+ref:
+  as: single-node-add-worker-2
+  from_image:
+    namespace: ocp
+    name: cli-jq
+    tag: latest
+  commands: single-node-add-worker-2-commands.sh
+  resources:
+    requests:
+      cpu: 100m
+      memory: 100Mi
+  documentation: |-
+    The Single Node Add Worker step adds a 2nd day worker to the SNO cluster under test. Step 2
```

Comment:
Let's just write in the description that it's identical to 1 but has to be its own step due to CI limitations